pkgs/development/python-modules/download/default.nix

```diff
@@ -0,0 +1,43 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+  wheel,
+  requests,
+  six,
+  tqdm,
+}:
+
+buildPythonPackage rec {
+  pname = "download";
+  version = "0.3.5";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-iEqIVHWzzb7Aqid+QWQ5lcM5Sh4GSggW9T//rko4ITA=";
+  };
+
+  build-system = [
+    setuptools
+    wheel
+  ];
+
+  dependencies = [
+    requests
+    six
+    tqdm
+  ];
+
+  pythonImportsCheck = [
+    "download"
+  ];
+
+  meta = {
+    description = "A quick module to help downloading files using python";
```

Comment:
Please ensure description meets guidelines. Remove the preposition

Reply:
I guess you mean article? I've removed it now.